lib/tests/misc.nix

```
@@ -269,6 +269,7 @@ runTests {
           strings
           possibly newlines
         '';
+        drv.outPath = "/path";
```

What happens to a path expr?
Does `__toString` work?

```suggestion
        drv.outPath = "/path";
        drv.foo = "ignore drv attr";
        pathValue = ./misc.nix;
        stringable.__toString = _: "hello toString";
        stringable.foo = "ignore stringable attr";
```